client/state/analytics/selectors.js

```diff
@@ -0,0 +1,7 @@
+export function isTracking( state ) {
+	return ( state.application.continuousTracking === 'TRACKING' );
+}
+
+export function isNotTracking( state ) {
+	return ( state.application.continuousTracking === 'NOT_TRACKING' );
+}
```

Comment:
No need for a dedicated selector for what should be equivalent to `! isTracking()` :slightly_smiling_face: